integrations/cmake/cmake_generator.rst

```diff
@@ -39,6 +39,14 @@ set global include directories, libraries directories, definitions, etc. so typi
 The ``conan_basic_setup()`` is split in smaller macros, that should be self explanatory. If you need to do
 something different, you can just use them individually.
 
+.. note::
+
+   You should consider using the Targets approach described below if you are using a recent enough cmake version.
```

Comment:
Maybe "described below" is a bit too much, but I would like to keep a sentence on what should be used instead. Otherwise, this might be mistaken for a recommendation against the cmake generator in general.

Reply:
Something like this:

```
This approach makes all dependencies visible to all CMake targets and may also
increase the build times due to unneeded include and library path components.
This is particularly relevant if you have multiple targets with different dependencies.
In that case, you should consider using the :ref:`gen_cmake_targets`.
```

Reply:
Sounds good, I will update my branch accordingly. Thanks!